src/ui/WebBrowser/Browser.js

```diff
@@ -1,87 +0,0 @@
-import { Linking } from 'react-native';
```

Comment:
This is another one that I think I mentioned earlier but we might need to bring these web browser files back. We need to be able to hide the nav on the web pages we browse too. We also pass the Rock token to some pages, but I'm guessing the core browser might do that now too.

Reply:
I say we query for the token and just attach it to the URL on the front-end

Reply:
Looks like you handled the nav hiding business on the links that needed it. And it looked like I was signed in on the links I need to be signed in on (giving, app feedback, etc.). Let's just go with this for now and if we run into something custom we need we can come back to it then.